components/brand/AboutUs.vue

```diff
@@ -1,53 +1,45 @@
 <template>
   <!-- About Us -->
   <div id="aboutUs" />
-  <section class="container mx-auto px-6 md:px-8 mt-12">
-    <div>
-      <img
-        src="~/assets/icons/about-us-egg-left.svg"
-        class="egg-left"
-      >
-    </div>
-    <div>
-      <img
-        class="purple-shape"
-        src="~/assets/images/purpleshape.svg"
-      >
-    </div>
-    <div class="flex flex-col lg:flex-row-reverse">
-      <div class="w-full lg:w-[500px] text-center md:text-left">
-        <h4 class="uppercase text-[32px] md:text-5xl text-primary font-semibold leading-tight">
+  <section class="container mx-auto px-6 md:px-8 lg:mt-12">
+    <div class="relative mb-6">
+      <div class="lg:absolute md:text-left lg:p-14">
+        <h4 class=" p-4 sm:p-8 lg:p-0 max-w-[240] lg:max-w-none absolute lg:static text-[32px] mb-6 md:text-5xl text-primary leading-tight font-bold">
           {{ pageData.fields.title }}
         </h4>
-        <p class="mb-4 mt-4 text-[20px] font-semibold text-primary">
-          {{ pageData.fields.subtitle }}
-        </p>
         <RichText
+          class="hidden lg:block font-semibold text-black text-xl "
           style="margin-top: 0; padding: 0;"
           :content="pageData.fields.paragraph"
         />
       </div>
+      
+      <img
+        class=" z-[-1] w-full lg:block"
+        src="~/assets/images/Illustration.png"
+      >
     </div>
+    <RichText
+      class="lg:hidden text-center lg:text-left block font-semibold text-black text-xl "
+      style="margin-top: 0; padding: 0;"
+      :content="pageData.fields.paragraph"
+    />
+    <div class="flex flex-col lg:flex-row-reverse" />
+ 
+    <h4 class="uppercase text-primary text-center font-semibold mb-6 lg:mb-8 mt-12 lg:mt-14 text-[32px] lg:text-[50px] leading-tight">
+      Our Mission
+    </h4>
+    <p class="text-black text-xl text-inter text-center lg:text-left font-['Inter'] ">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Eu non diam phasellus vestibulum lorem sed risus. Amet venenatis urna cursus eget nunc scelerisque. Volutpat lacus laoreet non curabitur gravida arcu ac tortor dignissim. Vitae et leo duis ut. Nisl suscipit adipiscing bibendum est ultricies integer. Diam quis enim lobortis scelerisque fermentum dui faucibus in ornare.
+    </p>
```

Comment:
I don't think we want to have this hard-coded. We should either fetch it from Contentful or hardcode the mission here.

Reply:
@wchesh24 let me know if you can add the mission field in contenful or if you have the text to hardcode it

Reply:
I have the field in Contentful now. Go ahead and fetch it from there. I don't have the right copy there but will update tomorrow when I get it from Jasmine or you can ask her directly and update contentful